lightning_pose/utils/plotting_utils.py

```diff
@@ -234,7 +361,7 @@ def make_predictions(
     n_frames_: int,  # total number of frames in the dataset or video
     batch_size: int,  # regular batch_size for images or sequence_length for videos
     data_name: str = "dataset",
-    save_folder: str = None
+    save_folder: str = None,
 ) -> Tuple[pd.DataFrame, Union[np.ndarray, None]]:
     """
```

Comment:
"""Predict poses on a single video from an already initialized model and data loader."""